### PR TITLE
Feature/phonefield detect country

### PR DESCRIPTION
**Pull request by mounir-ul**
## Description
Detect the country code from a given phone number, using the libphonenumber parseNumber method 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (new component, directive, filter or service)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Enhancement (non-breaking change which adds functionality such as new props, slots, event or method to an existing service)
- [ ] Refactoring/cleanup (non-breaking change that doesn't add any features but makes things better)
- [ ] Documentation/storybook (a change that do not affect any packages)

## How can this be tested
- [ ] Unit test (A new unit test as been created)
- [ ] Storybook (Indicate the name of the storybook to test the PR)
- [ ] Manual/Sandboxe (Indicate the name of the sandbox to test the PR)

## Links to issues

https://jira.dti.ulaval.ca/browse/AEL-1134

### CODE REVIEW

**Comment by jipigi** [comment]
Salut Mounir, merci pour ta contribution. 

Est-ce que tu pourrais ajouter un storybook pour tester quelques no étranger. Tu peux commencer par les 4 exemples que j'avais ajouté en commentaire  au [billet Jira](https://jira.dti.ulaval.ca/browse/AEL-1134?focusedCommentId=1316078&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1316078).
Il faudrait aussi décrire la section des releases notes pour documenter ton amélioration.

**Comment by mounir-ul** [comment]
Salut @jipigi merci!

J'ai ajouté la storybook, mais pour les releases notes je n'ai pas de breaking changes.

File: packages/modul-components/src/components/phonefield/phonefield.ts [hunk 1]
```diff
@@ -82,6 +82,12 @@ export class MPhonefield extends ModulVue {
     i18nCountryLabel: string = this.$i18n.translate('m-phonefield:country-label');
     i18nExample: string = this.$i18n.translate('m-phonefield:example');
 
+    beforeMount(): void {
```
Comment (chuckmah) [inline packages/modul-components/src/components/phonefield/phonefield.ts]: Etant donné le immediate: true  du watch tu peux enlever le beforeMount

Comment (mounir-ul) [reply packages/modul-components/src/components/phonefield/phonefield.ts]: Salut Charles, c'est fait!

**Pull request merged**